### ISSUE TITLE
Chatterino blocked users problem

### ISSUE THREAD

**Issue by MelloLuvsPeen** [Chatterino/chatterino2#3729]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

there is a limit of 100 blocked accounts that chatterino saves, when I restart chatterino, every account that Ive blocked other than the 100 that are already saved gets unblocked in chatterino but are still blocked on my twitch account. 

I am using the 7tv version but it did the same exact thing on 2.3.5

### OS and Chatterino Version

Chatterino 7.3.5 (commit 132b406cc) on Windows 10 Home 21H1 19043.1645

**Comment by iProdigy** [comment]
duplicate of #2767

**Comment by Felanbird** [comment]
Duplicate of #2767
